src/Microsoft.DotNet.Build.Tasks.Feed/SleetLogger.cs

```diff
@@ -43,7 +43,7 @@ public void LogDebug(string data)
 
         public void LogError(string data)
         {
-            _log.LogError(data);
+            _log.LogWarning(data);
```

Comment:
I think you should add some extra message here saying it was originally sent as an error, for diagnosability.